### PR TITLE
Fix issues #875 #764

### PR DESCRIPTION
**Pull request by fdodino**

### CODE REVIEW

File: org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend [hunk 1]
```diff
@@ -248,4 +248,13 @@ class NumberTestCase extends AbstractWollokInterpreterTestCase {
 		'''.interpretPropagatingErrors
 	}	
 
+	@Test
+	def void printString() {
+		'''program a {
+			assert.equals("4", 4.printString())
+			assert.equals("4.1", (4.1).printString())
+		}
+		'''.interpretPropagatingErrors
```
Comment (npasserini) [inline org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend]: Why not use .test() here?

File: org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend [hunk 2]
```diff
@@ -113,6 +113,10 @@ class WollokRepl {
 		println(obj?.toString.returnStyle)
 	}
 
+	def dispatch doPrintReturnValue(WollokObject wo) {
+		println(wo?.call("printString").toString.returnStyle)
+	}
+
 	def dispatch doPrintReturnValue(String obj) {
```
Comment (npasserini) [inline org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend]: I do not understand why is this necessary, can Strings be received here? Shouldn't we receive always a wollok object?

File: org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend (hunk 1 above)
Comment (fdodino) [inline org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend]: Because  I left it for another opportunity. :smile:

File: org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend (hunk 2 above)
Comment (fdodino) [inline org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend]: As Javi told me, Repl used dispatched method for Strings when Wollok used Java objects (previous to big Refactor). If you think it is a good choice I can remove def dispatch doPrintReturnValue(String obj)

**Comment by fdodino** [comment]
I realized I forgot to use System.lineSeparator instead of "\n" for Windows environments. Will be pushing another commits, so I can use test() instead of interpretPropagatingErrors()

File: org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend (hunk 2 above)
Comment (npasserini) [inline org.uqbar.project.wollok.launch/src/org/uqbar/project/wollok/launch/repl/WollokRepl.xtend]: @javierfernandes ?

File: org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend (hunk 1 above)
Comment (fdodino) [inline org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NumberTestCase.xtend]: Oh crap! I will add .test() because in another branch I have to modify all the tests also. I hope there are no conflicts.

**Comment by npasserini** [comment]
+1
